client/my-sites/upgrades/checkout/secure-payment-form.jsx

```diff
@@ -74,6 +78,17 @@ var SecurePaymentForm = React.createClass( {
 	},
 
 	render: function() {
+		if ( this.state.visiblePaymentBox === null ) {
+			return (
+				<EmptyContent
+					illustration='/calypso/images/drake/drake-500.svg'
+					title={ this.translate( 'Checkout is not available' ) }
+					line={ this.translate( "We're hard at work on the issue. Please check back shortly." ) }
+					action={ this.translate( 'Back to Plans' ) }
```

Comment:
I am open to ideas, I have used here the same as is on "domains not available" page.